vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridVirtualScrollingIT.java

```diff
@@ -33,11 +33,10 @@ public void gridScrolling() {
         waitForElementPresent(By.tagName("vaadin-grid"));
         WebElement grid = findElement(By.tagName("vaadin-grid"));
 
-        executeScript("arguments[0]._scrollToIndex(1000000);", grid);
+        executeScript("arguments[0].scrollToIndex(1000000);", grid);
 
         long firstVisible = (long) executeScript(
-                "return arguments[0]._firstVisibleIndex+arguments[0]._vidxOffset",
-                grid);
+                "return arguments[0]._firstVisibleIndex", grid);
```

Comment:
`_vidxOffset` is now a concept inside the virtualizer and doesn't concern grid. Now `_firstVisibleIndex` includes the offset value (it's still not public API!)